src/django_components/expression.py

```diff
@@ -1,50 +1,55 @@
-from typing import Any, Dict, List, Mapping, Optional, Union
+from typing import Any, Dict, List, Mapping, Optional, Tuple, Union
 
-from django.template import Context
+from django.template import Context, TemplateSyntaxError
 from django.template.base import FilterExpression, Parser
 
+Expression = Union[FilterExpression]
+RuntimeKwargsInput = Dict[str, Expression]
+RuntimeKwargPairsInput = List[Tuple[str, Expression]]
 
-class AggregateFilterExpression:
-    def __init__(self, dict: Dict[str, FilterExpression]) -> None:
-        self.dict = dict
 
+class RuntimeKwargs:
```

Comment:
Made the kwargs object into it's own class, so we can call `.resolve()` on it. This is where the processing of the `prop:key=val` syntax happens now.